stream-chat-android-ui-components/src/main/res/values/strings.xml

```diff
@@ -77,6 +77,9 @@
     <string name="stream_ui_send_label">Send</string>
     <string name="stream_ui_cancel_label">Cancel</string>
     <string name="stream_ui_attachment_downloading_started">Downloading started</string>
+    <string name="stream_ui_title_thread_reply">Thread Reply</string>
+    <string name="stream_ui_subtitle_thread_reply_single_user">with %s</string>
+    <string name="stream_ui_subtitle_thread_reply_many_users">with %d users</string>
```

Comment:
Shouldn't the multiple user / channel case look like this? Or is this another case that I didn't find in the design?

![image](https://user-images.githubusercontent.com/12054216/104451503-eed58b80-55a1-11eb-9f12-c555342740db.png)